port/osx/omrintrospect.c

```diff
@@ -64,11 +69,13 @@ static void resumeAllPreempted(PlatformWalkData *data);
 static int setupNativeThread(J9ThreadWalkState *state, thread_context *sigContext);
 static int suspendAllPreemptive(PlatformWalkData *data);
 static void upcallHandler(int signal, siginfo_t *siginfo, void *context);
+static int timedWait(int64_t seconds);
+static int64_t timeout(int64_t deadline);
 
 static void
 resumeAllPreempted(PlatformWalkData *data)
 {
-	if (data->cleanupRequired) {
+	if (TRUE == data->cleanupRequired) {
```

Comment:
This code can stay as-is since `cleanupRequired` is changed to a `BOOLEAN`.
```suggestion
	if (data->cleanupRequired) {
```

Reply:
Agreed. In my opinion, boolean values should only be used directly or compared with `FALSE/false` (never with `TRUE/true`:
* `if (flag)` ...
* `if (!flag)` ...
* `if (FALSE == flag)` ...

Reply:
I thought we wanted to be explicit with boolean constants